src/Onboarding.proto

```diff
@@ -185,3 +185,16 @@ message W4Event {
   google.protobuf.Timestamp deleted_at =16;
 }
 
+message CoreEmployeeOnboardingModuleEvent {
```

Comment:
Rename the event to `WSCoreEmployeeOnboardingEvent`

Reply:
Done